test/oli_web/controllers/open_and_free_controller_test.exs

```diff
@@ -210,6 +211,8 @@ defmodule OliWeb.OpenAndFreeControllerTest do
         end_date: DateTime.add(DateTime.utc_now(), 3600)
       })
 
+    insert(:section_project_publication, %{project: project, section: section, publication: publication})
```

Comment:
The manage source materials view is rendered inside the open and free section details.

As we are now rendering the view there, we need to insert a section_project_publication in the test otherwise it fails to find that information (which will always exist as a section is always linked to at least one base project or product). 

I don't know if my explanation was clear!  😅